lib/IDE/CodeCompletionResultBuilder.h

```diff
@@ -382,18 +391,16 @@ class CodeCompletionResultBuilder {
 #define KEYWORD(kw) .Case(#kw, true)
       shouldEscape = llvm::StringSwitch<bool>(Word)
 #include "swift/AST/TokenKinds.def"
-        .Default(false);
+                         .Default(Lexer::identifierMustAlwaysBeEscaped(Word));
```

Comment:
Nit: indentation

Reply:
This is what clang-format wants, apparently, because it's treated as a continuation of the expression `llvm::StringSwitch...` above.